README.md

```diff
@@ -22,34 +15,93 @@ To make this work, every application needs to do the following (at a minimum):
-
-## Getting started
-
-See the [ContainerPilot documentation](https://www.joyent.com/containerpilot/docs) to get started, or jump to specific sections:
-
-- [Installation in a container](https://www.joyent.com/containerpilot/docs/installation)
-- [Configuration](https://www.joyent.com/containerpilot/docs/configuration)
-- [Service discovery](https://www.joyent.com/containerpilot/docs/service-discovery)
-- [Container lifecycle](https://www.joyent.com/containerpilot/docs/lifecycle)
-- [Health checks](https://www.joyent.com/containerpilot/docs/health)
-- [`preStart`, `preStop`, `postStop`](https://www.joyent.com/containerpilot/docs/start-stop)
-- [Signals and operations](https://www.joyent.com/containerpilot/docs/signals)
-- [Periodic tasks](https://www.joyent.com/containerpilot/docs/tasks)
-- [Telemetry](https://www.joyent.com/containerpilot/docs/telemetry)
-- [Coprocesses](https://www.joyent.com/containerpilot/docs/coprocesses)
-- [Frequently asked questions](https://www.joyent.com/containerpilot/docs/faq)
-- [Support](https://www.joyent.com/containerpilot/docs/support)
+ContainerPilot is an init system designed to live inside the container. It acts as a process supervisor, reaps zombies, run health checks, registers the app in the service catalog, watches the service catalog for changes, and runs your user-specified code at events in the lifecycle of the container to make it all work right. ContainerPilot uses Consul to coordinate global state among the application containers.
+
+## Quick Start Guide
+
+Check out our ["Hello, World" application](https://github.com/autopilotpattern/hello-world) on GitHub. Assuming you have Docker and Docker Compose available, it's as easy as:
+
+```
+git clone git@github.com:autopilotpattern/hello-world.git
+cd hello-world
+docker-compose up -d
+open http://localhost
+```
+
+This application blueprint demonstrates using ContainerPilot to update Nginx upstream configuration at runtime. Try scaling up via `docker-compose scale hello=2 world=3` to see the Nginx configuration updated.
+
+You can also [download](https://github.com/joyent/containerpilot/releases) the latest release of ContainerPilot from GitHub.
+
+## Documentation
+
+Documentation for ContainerPilot and where it fits with the rest of the Triton ecosystem can be found at [www.joyent.com/containerpilot](https://www.joyent.com/containerpilot). The index below links to the documentation in this repo for convenience.
+
+[Lifecycle](./docs/10-lifecycle.md)
```

Comment:
The intent is that the Lifecycle section is the high-level view of only the core concepts work, whereas the Configuration section is the nitty-gritty. I'm also making the assumption that people don't read the whole documentation (alas!) so redundancy is ok.